tfjs-backend-webgpu/src/setup_test.ts

```diff
@@ -66,55 +66,63 @@ const TEST_FILTERS: TestFilter[] = [
   {
     startsWith: 'cos ',
     excludes: [
-      'gradients', // Failing on MacOS
-      'gradient with clones', // Failing on MacOS
+      'gradients',             // Failing on MacOS
+      'gradient with clones',  // Failing on MacOS
     ],
   },
   {
     startsWith: 'tan ',
     excludes: [
-      'gradients', // Failing on MacOS
+      'gradients',  // Failing on MacOS
       //'gradient with clones', // Failing on MacOS
     ],
   },
   {
     startsWith: 'acosh ',
     excludes: [
-      'propagates NaNs', // Failing on MacOS
-      'gradient with clones', // Failing on MacOS
+      'propagates NaNs',       // Failing on MacOS
+      'gradient with clones',  // Failing on MacOS
     ],
   },
   {
     startsWith: 'asinh ',
     excludes: [
-      'propagates NaNs', // Failing on MacOS
+      'propagates NaNs',  // Failing on MacOS
       //'gradient with clones', // Failing on MacOS
     ],
   },
   {
     startsWith: 'atanh ',
     excludes: [
-      'propagates NaNs', // Failing on MacOS
+      'propagates NaNs',  // Failing on MacOS
       //'gradient with clones', // Failing on MacOS
     ],
   },
   {
     startsWith: 'sigmoid ',
     excludes: [
-      'propagates NaNs', // Failing on MacOS
+      'propagates NaNs',  // Failing on MacOS
       //'gradient with clones', // Failing on MacOS
     ],
   },
   {
     startsWith: 'unsortedSegmentSum ',
     excludes: [
-      'ignores negative segmentIds', // Failing on MacOS
+      'ignores negative segmentIds',  // Failing on MacOS
     ],
   },
   {
     startsWith: 'log ',
     excludes: [
-      'log propagates NaNs', // Failing on MacOS
+      'log propagates NaNs',  // Failing on MacOS
+    ],
+  },
+
+  {
+    startsWith: 'softmax ',
+    excludes: [
+      'underflow',        // Actual: NaN,NaN. Expected: 0.5,0.5.
```

Comment:
Why is this failing?

Reply:
Fix it here https://github.com/tensorflow/tfjs/pull/7588/commits/7a08da099a67ca1a5a539e0ca090c4b750417c94. Thanks.